src/hotspot/share/runtime/os.hpp

```diff
@@ -225,22 +225,23 @@ class os: AllStatic {
   static bool   pd_uncommit_memory(char* addr, size_t bytes, bool executable);
   static bool   pd_release_memory(char* addr, size_t bytes);
 
-  static char*  pd_attempt_map_memory_to_file_at(char* addr, size_t bytes, int file_desc);
+  static char*  pd_attempt_map_memory_to_file_at(char* addr, size_t bytes, int file_desc, MEMFLAGS flag);
 
   static char*  pd_map_memory(int fd, const char* file_name, size_t file_offset,
-                           char *addr, size_t bytes, bool read_only = false,
+                           char *addr, size_t bytes,
+                           MEMFLAGS flag,
+                           bool read_only = false,
```

Comment:
Mixes param layout style. (Plus earlier comment that the default values should probably be removed so that MEMFLAGS can be put last).